lib/maxmind/geoip2/record/traits.rb

```diff
@@ -131,6 +129,28 @@ def legitimate_proxy?
           get('is_legitimate_proxy')
         end
 
+        # The {https://en.wikipedia.org/wiki/Mobile_country_code mobile country
+        # code (MCC)} associated with the IP address and ISP.
+        #
+        # This attribute is only available from the City and Insights web service
+        # and the GeoIP2 Enterprise database.
```

Comment:
Does it make sense to have this documentation here? Same with MNC.

Reply:
Hmm, right. Maybe not. This is a straight copy though other than the copyright header. We've got it for the other similar traits too. I'm kind of inclined to leave it to avoid making it a fork.

Reply:
Ah, ok. Yeah, this won't matter for long anyway given the other story.